motr/client.h

```diff
@@ -554,11 +554,11 @@ enum m0_idx_opcode {
 	M0_IC_PUT,                 /* 16 */
 	/** Delete the value, if any, for the given key. */
 	M0_IC_DEL,                 /* 17 */
-	/** Given a key, return the next key and its value. */
+	/** Given a key, return the next keys and their values. */
 	M0_IC_NEXT,                /* 18 */
-	/** Check an index for an existence. */
+	/** Check index for the given key existence. */
 	M0_IC_LOOKUP,              /* 19 */
-	/** Given an index id, get the list of next indices. */
+	/** Given a key, return the list of next keys. */
```

Comment:
I don't think this is correct.

This operation is to enumerate indices, not the records of an index.
It is like "getdir()", from this index, to list a predefined number of indices next to this one.

Reply:
Fixed.

Reply:
I still think the original document is better.

Reply:
For which op?

Reply:
M0_IC_LIST

Reply:
Sorry, but it was just wrong! `M0_IC_LIST` returns the list of keys from an index (given the starting point key), not the list of indexes.

Reply:
No, this is wrong.

You can refer to this: https://github.com/Seagate/cortx-motr/pull/1651
It has some example output of "index list".

Reply:
I see...

Unfortunately, this PR was already merged by accident, so here is the fix - https://github.com/Seagate/cortx-motr/pull/1653.